aptos-move/framework/aptos-token/sources/TokenCoinSwap.move

```diff
@@ -2,14 +2,14 @@
 /// 1. hold tokens escrow to prevent token been transferred
 /// 2. list token for swapping with a targeted CoinType.
 /// 3. execute the swapping
-module AptosFramework::TokenCoinSwap {
+module AptosToken::TokenCoinSwap {
     use Std::Event::{Self, EventHandle};
     use Std::Signer;
     use AptosFramework::Table::{Self, Table};
-    use AptosFramework::TokenV1::{Self, Token, TokenId, deposit_token, withdraw_token, merge, split};
     use AptosFramework::Coin;
     use AptosFramework::Timestamp;
     use AptosFramework::TypeInfo::{Self, TypeInfo};
+    use AptosToken::TokenV1::{Self, Token, TokenId, deposit_token, withdraw_token, merge, split};
```

Comment:
I suggest to not yet use the versions in the name. We may have major versions at the point of time you want to make a breaking change. If not, it is totally legit and common practice to name this `Token`, `TokenV2`, `TokenV3`.

Reply:
Yes, will change the name back to Token.move once the migration is done. Temporarily use TokenV1 so that I can iterate on the token modules separately